x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/inspect/inspect.spec.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
+import { ARCHIVER_ROUTES } from '../constants/archiver';
+import archives_metadata from '../constants/archives_metadata';
+
+export default function inspectFlagTests({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const apmApiClient = getService('apmApi');
```

Comment:
Does this apmApiClient need any cleanup in an `after` method?  
Perhaps that's already occuring?

Reply:
As far as I know, it’s not necessary. 
This service acts as a gateway for API requests.

It’s used in nearly all our test files, and we use it in the same way across them.

Reply:
`apmApi` doesn't need any clean up.